packages/nextjs/components/Processes/RecoveryProcess/RecoveryProcess.tsx

```diff
@@ -139,15 +154,57 @@ export const RecoveryProcess = ({
       <CustomPortal
         title={"Your assets have been recovered!"}
         description={
-          "If we have helped you please, share the tool and if you want let us a tip to continue contributing"
+          "Check your safe wallet for your retrieved assets. Share your journey and consider to support us with a tip to continue serving the crypto community."
```

Comment:
"Check your safe wallet for your retrieved assets. If you were helped by this tool, please consider sharing this on social media and if you are feeling generous, consider giving us a small tip so we can continue providing helpful tools to the crypto community."